src/main/java/org/apache/commons/codec/digest/MurmurHash3.java

```diff
@@ -393,7 +401,7 @@ public static long hash64(final byte[] data, final int offset, final int length,
 	 * @return - 128 bit hash (2 longs)
 	 */
 	public static long[] hash128(final String data) {
-		final byte[] origin = data.getBytes();
+		final byte[] origin = data.getBytes(StandardCharsets.UTF_8);
```

Comment:
Why the charset conversion? Changing away from using the default charset is not related to the other changes in this PR.